intro.tex

```diff
@@ -0,0 +1,19 @@
+\section{Introduction}
+
+
+In the following document we will explain step by step the current procedure to upgrade the version of
```

Comment:
This sentence needs to be unwrapped.

Reply:
same with this text, removed the newlines